rfcs/rfc-covid-agm-extension.md

```diff
@@ -0,0 +1,48 @@
+# RFC For COVID AGM date extension
+
+- Start Date: (2021-01-19)
+- Target Major Version: (None)
+- Reference Issues: (None)
+- Entity Issue: (https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6198)
+- Implementation PR: (None)
+
+# Summary
+
+The registrar gave all co-ops an extension on their AGMs for COVID. We need to enable them to select those extended dates in the AR with the AGM date picker. It currently restricts them to Jan 1st - Dec 31st of the calendar year.
+
+First, we need to update the default date picker validation, and allow an AGM date anywhere from Jan 1st to April 30th of the next year. Currently it is Jan 1st to Dec 31st of the same year. This is because many co-ops filing their 2020 AR hold the AGM for that year in 2021
+
+Our solution for the COVID AGM extensions is to have them check a checkbox that they are taking the blanket extension. When they check the checkbox, validation on the date picker will go from the default up to April 30th of the next year, to 6 months ahead (October 31st of the next year).
```

Comment:
Change
"will go from the default up to April 30th of the next year, to"
to
"will go from the default (April 30th of the next year), to".